packages/flame/lib/src/sprite_batch.dart

```diff
@@ -17,6 +17,7 @@ extension SpriteBatchExtension on Game {
     BlendMode defaultBlendMode = BlendMode.srcOver,
     RSTransform? defaultTransform,
     Images? imageCache,
+    bool? applyFlip,
```

Comment:
What about calling this parameter `allowFlips` ?

Reply:
That's a great idea! thanks